src/test/java/org/broadinstitute/consent/http/resources/SupportResourceTest.java

```diff
@@ -0,0 +1,189 @@
+    try (Response response = supportResource.postRequest(body)) {
+      assertEquals(HttpStatusCodes.STATUS_CODE_BAD_REQUEST, response.getStatus());
+    }
+  }
+
+  @Test
+  void testPostRequestInvalidEmail() {
+    String body = """
+        {
+          "name": "Test User",
+          "subject": "Test Subject",
+          "description": "Test Description",
+          "type": "QUESTION",
+          "url": "https://example.com",
+          "uploads": [
+            "token1",
+            "token2",
+          ]
+        }
+        """;
+    try (Response response = supportResource.postRequest(body)) {
+      assertEquals(HttpStatusCodes.STATUS_CODE_BAD_REQUEST, response.getStatus());
+    }
+  }
+
+  @Test
+  void testPostRequestInvalidSubject() {
+    String body = """
+        {
+          "name": "Test User",
+          "email": "test.user@example.com",
+          "description": "Test Description",
+          "type": "QUESTION",
+          "url": "https://example.com",
+          "uploads": [
+            "token1",
+            "token2",
+          ]
+        }
+        """;
+    try (Response response = supportResource.postRequest(body)) {
+      assertEquals(HttpStatusCodes.STATUS_CODE_BAD_REQUEST, response.getStatus());
+    }
+  }
+
+  @Test
+  void testPostRequestInvalidDescription() {
+    String body = """
+        {
+          "name": "Test User",
+          "email": "test.user@example.com",
+          "subject": "Test Subject",
+          "type": "QUESTION",
+          "url": "https://example.com",
+          "uploads": [
+            "token1",
+            "token2",
+          ]
+        }
+        """;
+    try (Response response = supportResource.postRequest(body)) {
+      assertEquals(HttpStatusCodes.STATUS_CODE_BAD_REQUEST, response.getStatus());
+    }
+  }
+
+  @Test
+  void testPostRequestInvalidType() {
+    String body = """
+        {
+          "name": "Test User",
+          "email": "test.user@example.com",
+          "subject": "Test Subject",
+          "description": "Test Description",
+          "type": "type",
+          "url": "https://example.com",
+          "uploads": [
+            "token1",
+            "token2",
+          ]
+        }
+        """;
+    try (Response response = supportResource.postRequest(body)) {
+      assertEquals(HttpStatusCodes.STATUS_CODE_BAD_REQUEST, response.getStatus());
+    }
+  }
+
+  @Test
+  void testPostRequestInvalidURL() {
+    String body = """
+        {
+          "name": "Test User",
+          "email": "test.user@example.com",
+          "subject": "Test Subject",
+          "description": "Test Description",
+          "type": "QUESTION",
+          "uploads": [
+            "token1",
+            "token2",
+          ]
+        }
+        """;
+    try (Response response = supportResource.postRequest(body)) {
+      assertEquals(HttpStatusCodes.STATUS_CODE_BAD_REQUEST, response.getStatus());
+    }
+  }
+
+  @Test
+  public void testPostUpload() throws Exception {
```

Comment:
`public` is unnecessary here
```suggestion
  void testPostUpload() throws Exception {
```